tests/cache/tests.py

```diff
@@ -1398,6 +1398,20 @@ def test_pylibmc_options(self):
         self.assertTrue(cache._cache.binary)
         self.assertEqual(cache._cache.behaviors['tcp_nodelay'], int(True))
 
+    def test_default(self):
+        """
+        Test if default value is used when it's provided. Tests if None is
+        returned instead of the default, if it has been explicitly saved to the
+        cache.
+        """
+        default = "example default value"
```

Comment:
Use single quotes (unless a string contains a single quote) as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

Reply:
:+1: